docs/faq/security.md

```diff
@@ -7,12 +7,16 @@ title: Security FAQ
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/faq/security"/>
 </head>
 
-**Is there a Hardening Guide?**
+### Is there a Hardening Guide?
 
 The Hardening Guide is now located in the main [Security](../reference-guides/rancher-security/rancher-security.md) section.
 
-<br/>
-
-**What are the results of Rancher's Kubernetes cluster when it is CIS benchmarked?**
+### What are the results of Rancher's Kubernetes cluster when it is CIS benchmarked?
```

Comment:
This heading is written kind of awkwardly but maybe we can look into fixing that in another PR. I did notice it but couldn't think of a decent way to rephrase